apps/examples/wc-next10/pages/index.jsx

```diff
@@ -4,13 +4,18 @@ import dynamic from 'next/dynamic';
 const Counter = dynamic(() => import('../Counter'), { ssr: false });
 const VariantSwitch = dynamic(() => import('../VariantSwitch'), { ssr: false });
 const ButtonSizes = dynamic(() => import('../ButtonSizes'), { ssr: false });
+const PieCookieBanner = dynamic(
+    () => import('@justeattakeaway/pie-cookie-banner/dist/react').then(mod => mod.PieCookieBanner),
```

Comment:
I'm curious to know if this works, or whether you already tried it and then had to use `.then`?
```suggestion
const { PieCookieBanner } = dynamic(
    () => import('@justeattakeaway/pie-cookie-banner/dist/react'),
```

Reply:
I don't think this will work as we need to `await` for the promise. 
i followed the next [docs](transformIgnorePatterns) here in case it helps.